src/nvector/petsc/nvector_petsc.c

```diff
@@ -415,9 +415,9 @@ void N_VSetArrayPointer_Petsc(sunrealtype *v_data, N_Vector v)
   return;
 }
 
-void *N_VGetCommunicator_Petsc(N_Vector v)
+MPI_Comm N_VGetCommunicator_Petsc(N_Vector v)
```

Comment:
PETSc can be built without MPI, so should this be `SUNComm` instead, or do we require that we only link against MPI-enabled PETSc installations?

Reply:
We require that PETSc is built wiht MPI-enabled.